subprojects/build-scan-performance/src/testFixtures/groovy/org/gradle/performance/AbstractBuildScanPluginPerformanceTest.groovy

```diff
@@ -52,7 +58,15 @@ class AbstractBuildScanPluginPerformanceTest extends AbstractPerformanceTest {
     @Shared
     String pluginVersionNumber = resolvePluginVersion()
 
+    GradleDistribution distribution
+
     def setup() {
+        def baselineVersions = resolveBaselineVersions()
```

Comment:
If this doesn't cause any problems then I think this is fine. Though if this code runs and fails the build even though you didn't run any of the affected performance tests, then I suppose this should be moved to the time when the test actually runs, like we do for `toBaselineVersions`.

Reply:
pls check if that's what you meant

Reply:
It is not. I meant moving this to the `*Runner`, though like I said, if this doesn't cause any problems let's leave it as is.